pom.xml

```diff
@@ -15,54 +15,73 @@
   <parent>
     <groupId>com.google.cloud</groupId>
     <artifactId>google-cloud-shared-config</artifactId>
-    <version>1.5.4</version>
+    <version>1.5.5</version>
   </parent>
 
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
 
-    <kafka.version>3.3.1</kafka.version>
```

Comment:
Why are we getting rid of version properties? This is the recommended POM structure for GA libraries I was told.

Reply:
In general, can you separate out POM changes you are making for this change and the ones that are purely stylistic? Or are all these changes needed as-is?

Reply:
This also isn't the recommended way to define dependency versions for shared dependencies- I'm converting this to follow the best practice.

Reply:
Discussed offline, this now uses BOM structure - https://cloud.google.com/java/docs/bom.